src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -613,15 +613,16 @@ public function isAbsolutePath($file)
      * @param string $dir    The directory where the temporary filename will be created
      * @param string $prefix The prefix of the generated temporary filename
      *                       Note: Windows uses only the first three characters of prefix
+     * @param string $suffix The suffix of the generated temporary filename
      *
      * @return string The new temporary filename (with path), or throw an exception on failure
      */
-    public function tempnam($dir, $prefix)
+    public function tempnam($dir, $prefix, string $suffix = '')
```

Comment:
Adding an additional argument is still a BC break

```suggestion
    public function tempnam($dir, $prefix/*, string $suffix = ''*/)
    {
        $suffix = \func_num_args() > 2 ? func_get_arg(2) : '';

```